src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
```

Comment:
Please use constants with proper naming instead of magic values to make code more readable and flexible